content/manager_webui/blueprints-page.md

```diff
@@ -12,7 +12,36 @@ relationships_link: dsl-spec-relationships.html
 ---
 {{% gsSummary %}}{{% /gsSummary %}}
 
-When clicking on the `Blueprints` tab and choosing a blueprint you will be able to choose one of the following:
+When clicking on the `Blueprints` tab you are presented with the list of blueprints.<br>
+![Blueprints index]({{<img "ui/blueprintsPage/index.png">}})<br>
+The table presents each blueprint's Name, Creation date, Update Date, Number of deployments and a dropdown menu with the available actions to perform on the blueprint.<br>
+![Blueprints actions]({{<img "ui/blueprintsPage/actions.png">}})
+
+### Upload a blueprint
+If you haven't uploaded any blueprint, you can do so by clicking the 'Upload Blueprint' button.
+![upload blueprint dialog]({{<img "ui/blueprintsPage/uploadBlueprint.png">}})<br>
+The button will open a dialog containing a form requiring the blueprint archive(local blueprint archive or a remote link), name and the blueprint filename within the archive. If blueprint filename field omitted it defaults to a blueprint.yaml file placed in the top level directory.<br>
```

Comment:
If the blueprint filename field is omitted

blueprint.yaml -> add backticks please so that users know the name of the file is "blueprint.yaml"